File: AddPost/resultFormatter.js
module.exports = {
    getResultSingle: function (obj) {
        var array=[];
        array.push(obj)
        var result = {
            "success":true,
            "result":array,
            "error":null
        };
        return result;
    },
    getResultMultiple: function (obj) {
        var result = {
            "success":true,
            "result":obj,
            "error":null
        };
        return result;
    },
    getReultError: function (msg) {
        var result = {
          "success":false,
          "result":null,
          "error":msg
        };
        return result;
    }
};

File: SearchUser/resultFormatter.py
def getResultSingle(obj):
    array = []
    array.append(obj)
    result = {
        "success":True,
        "result":array,
        "error":None
    }
    return result

def getResultMultiple():
    result = {
        "success":True,
        "result":obj,
        "error":None
    }
    return result

def getResultError(msg):
    result = {
      "success":False,
      "result":None,
      "error":msg
    }
    return result
